handshake/connection_parameters_manager_test.go

```diff
@@ -290,7 +290,7 @@ var _ = Describe("ConnectionsParameterManager", func() {
 					TagMIDS: {3, 0, 0, 0},
 				})
 				Expect(err).ToNot(HaveOccurred())
-				Expect(cpm.GetMaxIncomingStreams()).To(BeNumerically(">", protocol.MaxStreamsPerConnection))
+				Expect(cpm.GetMaxIncomingStreams()).To(BeNumerically(">", maxStreamsInTest))
```

Comment:
I fixed this based on the code it is supposed to check. If that was incorrect, this might be too